cmd/healthcheck/main_test.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestHealthStatusHealthy(t *testing.T) {
+	if os.Getenv("FLAG") == "1" {
+		healthCheckDefaultUrl := "127.0.0.1:13133"
```

Comment:
nit: this is the endpoint no? should it be `healthCheckDefaultEndpoint`?

Reply:
Thanks. Updated.